GitUp/Application/en.lproj/MainMenu.xib

```diff
@@ -594,7 +594,7 @@ You must close and reopen any opened repositories in GitUp after changing this s
                                 </connections>
                             </menuItem>
                             <menuItem isSeparatorItem="YES" id="qQz-Pc-UPf"/>
-                            <menuItem title="Browse HEAD Ancestors…" keyEquivalent="d" id="6tA-ab-KI4">
+                            <menuItem title="Browse HEAD Commits…" keyEquivalent="d" id="6tA-ab-KI4">
```

Comment:
I'm not sure "Browse HEAD Commits" make sense. There is only one HEAD commit by definition. The original label is still better after thinking about it.